### PR TITLE
Integrated dotenv for mysql credendtials

### PR DESCRIPTION
**Pull request by turbat-bay**
Resolves issue https://github.com/ACM-VIT/dolos/issues/40

### CODE REVIEW

File: src/database.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  user: "root",
+  password: "root"
+};
+
+if (config.parsed) {
+  connectionString.user = config.parsed.DB_USER;
+  connectionString.password = config.parsed.DB_PASS;
+}
+
+var con = mysql.createConnection(connectionString);
```
Comment (YashKumarVerma) [inline src/database.js:15]: use OR operator as the same can be done in short.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: src/database.js (hunk 1 above)
Comment (turbat-bay) [reply src/database.js]: thanks for the feedback!

File: src/database.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 var mysql = require('mysql');
+var config = require('dotenv').config();
 
-var con = mysql.createConnection({
+var connectionString = {
   host: "localhost",
```
Comment (YashKumarVerma) [inline src/database.js:5]: why aren't we loading the host from env ?

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by YashKumarVerma — APPROVED** [review]
approving as required changes done, but PR sent before this one for the same issue would be merged.

**Pull request closed without merging**